src/pal/inc/pal.h

```diff
@@ -1889,6 +1889,23 @@ typedef struct _CONTEXT {
 // support any other values in the ExtendedRegisters) but we might as well be as accurate as we can.
 #define CONTEXT_EXREG_XMM_OFFSET 160
 
+typedef struct _KNONVOLATILE_CONTEXT_POINTERS {
+
+    // TODO WIP x86/Linux, need to fix this.
+    PDWORD Eax;
+    PDWORD Ebx;
+    PDWORD Ecx;
+    PDWORD Edx;
+    PDWORD Esi;
+    PDWORD Edi;
+    PDWORD Ebp;
+    PDWORD Esp;
+    PDWORD Eip;
```

Comment:
There is more that do not belong here - there should only be: Ebx, Esi, Edi, Ebp.